wger - update script failed

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

wger

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/wger.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Hi there, 
The WGER update to v2.3 encountered an error during execution. 
`   
  ✔️   Stopped wger
⠋ Updating wger to v2.3environment: line 38: cd: /tmp/tmp.RSpw5b6jqq: Not a directory
⠙ Updating wger to v2.3
[ERROR] in line 38: exit code 0: while executing command cd $temp_file`

Please note that the temporary file at /tmp/tmp.RSpw5b6jqq

Thanks for your help

### 🔄 Steps to reproduce the issue.

type update in the wger lxc console

### ❌ Paste the full error output (if available).

```
environment: line 38: cd: /tmp/tmp.RSpw5b6jqq: Not a directory
[ERROR] in line 38: exit code 0: while executing command cd $temp_file
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/wger.sh
+++ b/ct/wger.sh
@@ -20,46 +20,45 @@ color
 catch_errors
 
 function update_script() {
-    header_info
-    check_container_storage
-    check_container_resources
-    if [[ ! -d /home/wger ]]; then
-        msg_error "No ${APP} Installation Found!"
-        exit
-    fi
-    RELEASE=$(curl -fsSL https://api.github.com/repos/wger-project/wger/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3)}')
-    if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
-        msg_info "Stopping $APP"
-        systemctl stop wger
-        msg_ok "Stopped $APP"
-
-        msg_info "Updating $APP to v${RELEASE}"
-        temp_file=$(mktemp)
-        cd $temp_file
-curl -fsSL "https://github.com/wger-project/wger/archive/refs/tags/$RELEASE.tar.gz" -o "$temp_file"
-        tar xzf $temp_file
-        cp -rf wger-$RELEASE/* /home/wger/src
-        cd /home/wger/src
-        python3 manage.py migrate &>/dev/null
-        yarn install &>/dev/null
-        yarn build:css:sass &>/dev/null
-        python3 manage.py collectstatic --noinput &>/dev/null
-        echo "${RELEASE}" >/opt/${APP}_version.txt
-        msg_ok "Updated $APP to v${RELEASE}"
-
-        msg_info "Starting $APP"
-        systemctl start wger
-        msg_ok "Started $APP"
-
-        msg_info "Cleaning Up"
-        rm -rf $temp_file
-        msg_ok "Cleanup Completed"
-
-        msg_ok "Update Successful"
-    else
-        msg_ok "No update required. ${APP} is already at v${RELEASE}"
-    fi
+  header_info
+  check_container_storage
+  check_container_resources
+  if [[ ! -d /home/wger ]]; then
+    msg_error "No ${APP} Installation Found!"
     exit
+  fi
+  RELEASE=$(curl -fsSL https://api.github.com/repos/wger-project/wger/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3)}')
+  if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
+    msg_info "Stopping $APP"
+    systemctl stop wger
+    msg_ok "Stopped $APP"
+
+    msg_info "Updating $APP to v${RELEASE}"
+    temp_file=$(mktemp)
+    curl -fsSL "https://github.com/wger-project/wger/archive/refs/tags/$RELEASE.tar.gz" -o "$temp_file"
+    tar xzf "$temp_file"
+    cp -rf wger-"$RELEASE"/* /home/wger/src
+    cd /home/wger/src
+    python3 manage.py migrate &>/dev/null
+    yarn install &>/dev/null
+    yarn build:css:sass &>/dev/null
+    python3 manage.py collectstatic --noinput &>/dev/null
+    echo "${RELEASE}" >/opt/${APP}_version.txt
+    msg_ok "Updated $APP to v${RELEASE}"
+
+    msg_info "Starting $APP"
+    systemctl start wger
+    msg_ok "Started $APP"
+
+    msg_info "Cleaning Up"
+    rm -rf "$temp_file"
+    msg_ok "Cleanup Completed"
+
+    msg_ok "Update Successful"
+  else
+    msg_ok "No update required. ${APP} is already at v${RELEASE}"
+  fi
+  exit
 }
 
 start

--- a/install/wger-install.sh
+++ b/install/wger-install.sh
@@ -15,10 +15,10 @@ update_os
 
 msg_info "Installing Dependencies"
 $STD apt-get install -y \
-    git \
-    gnupg \
-    apache2 \
-    libapache2-mod-wsgi-py3
+  git \
+  gnupg \
+  apache2 \
+  libapache2-mod-wsgi-py3
 msg_ok "Installed Dependencies"
 
 msg_info "Installing Python"

